2009-06-20  Carlos O'Donell  <carlos@codesourcery.com>

	* Versions.def: Add GLIBC_2.10 for libpthread.

--- a/Versions.def
+++ b/Versions.def
@@ -89,6 +89,7 @@ libpthread {
   GLIBC_2.3.4
   GLIBC_2.4
   GLIBC_2.6
+  GLIBC_2.10
   GLIBC_PRIVATE
 }
 libresolv {
